src/register_types.h

```diff
@@ -0,0 +1,11 @@
+#ifndef FMOD_REGISTER_TYPES_H
+#define FMOD_REGISTER_TYPES_H
+
+#include <godot_cpp/core/class_db.hpp>
```

Comment:
`godot_cpp` should not be part of imports. Include paths seems to be wrongly configured.

Reply:
Interesting. I did this because all other GDExtension projects do the same. Will change this.

Reply:
IMO we should not add the godot_cpp because extensions are meant to code as in engine module.